Source/WebCore/page/Navigation.cpp

```diff
@@ -424,7 +424,7 @@ bool Navigation::innerDispatchNavigateEvent(NavigationNavigationType navigationT
         downloadRequestFilename,
         info,
         canIntercept,
-        false, // FIXME: userInitiated
+        userInitiated,
```

Comment:
Instead of passing this down as a parameter can we just query the document here in this functiion?

Reply:
That is a good suggestion,I did not consider that. Will upload the new patch tomorrow, thanks!

Reply:
Done, but also corrected the used API, i.e. hasRecentUserInteractionForNavigationFromJS was the wrong one.